apps/src/templates/sectionAssessments/StudentsAssessmentsMCTable.jsx

```diff
@@ -101,7 +100,7 @@ class StudentsAssessmentsMCTable extends Component {
   submissionTimestampColumnFormatter = (submissionTimeStamp, {rowData}) => {
     const submissionStatus = rowData.submissionStatus;
 
-    if (submissionStatus === 'Completed') {
+    if (submissionStatus === true) {
```

Comment:
`if (submissionStatus) {` <-- booleans are truthy so you can write it this way

Reply:
Fixed.  Also set submissionStatus == 'In progress'